python/ccf/clients.py

```diff
@@ -409,6 +413,16 @@ def request(
 
         auth_value = None
         if self.signing_auth is not None:
+            # Add content length of 0 when signing a GET request
+            if request.http_verb == "GET":
+                if "Content-Length" in extra_headers and extra_headers[
+                    "Content-Length"
+                ] != str(0):
+                    raise ValueError(
+                        "Content-Length should be set to 0 for GET requests"
+                    )
+                else:
+                    extra_headers["Content-Length"] = str(0)
```

Comment:
```suggestion
                    extra_headers["Content-Length"] = "0"
```

Always use literals where possible, they're not only faster to execute (no lookup), but also can't be shadowed (no lookup!).